pubspec.yaml

```diff
@@ -6,16 +6,14 @@ description: Words guess game app
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
 # The following defines the version and build number for your application.
-# A version number is three numbers separated by dots, like 1.2.43
-# followed by an optional build number separated by a +.
```

Comment:
Simplify the version extraction regex for now by quick and dirty fix: removing these lines 🤣 so the first semantic number detected is the one below... this could be improved later because I'm going to create the issue !